## Possible Options

This is the error now. [(error)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=705254&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=679)

```sh
import: 'nlptest'
import: 'nlptest'
+ pip check
nlptest 1.2.0 requires rouge-score, which is not installed.
Tests failed for nlptest-1.2.0-pyhd8ed1ab_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for nlptest-1.2.0-pyhd8ed1ab_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
TESTS FAILED: nlptest-1.2.0-pyhd8ed1ab_0.conda
##[error]Bash exited with code '1'.
```

This package (`rouge-score`) was missing from run:requirements.

> :bell: I have updated the recipe and added `rouge-score`. The new build should work, unless something else is missing.

cc: @luca-martial @ArshaanNazir

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -33,6 +33,7 @@ requirements:
     - openai
     - evaluate
     - inflect
+    - rouge-score
 
 test:
   imports:

